docs/dymos_book/examples/cannonball_implicit_duration/cannonball_implicit_duration.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>The bit about setting up two phases and linking them accordingly needs to be changed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/dymos/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>